include/menu.h

```diff
@@ -38,6 +39,10 @@ enum {
     GZ_KORL_FLAG_MENU,
     GZ_SAVEWARP_FLAG_MENU,
 
+    /* Scene sub-menus */
+    GZ_DUNGEON_MENU,
+ //   GZ_PLACEHOLDER2_MENU,

```

Comment:
Why is this here? Does this affect your flag-adding script?

Reply:
the dungeon menu was once in the sceenes menu. This is a leftover. Now I understand why dark link is still called torch2, it is annoying going back and remembering to change all these little things but i will now.